### ISSUE TITLE
Installation / Readme informations

### ISSUE THREAD

**Issue by piercus** [kazuto1011/c3d_multilayer#1]
Hello,

I'm interested in your project but there is no documentation.
Can you please give more informations on how to install/ what are the performances. 

Is your model accurate after training ?

Thank for your feedabck

**Comment by kazuto1011** [comment]
Hi, @piercus.
Thank you for your interest and comments. Added some project description to README.md. This is my trial project for learning some machine learning tools ([Chainer](https://github.com/pfnet/chainer) and [TensorFlow](https://github.com/tensorflow/tensorflow) for now) and I've tried transfer learning. Trains a 3-layer perceptron giving [pre-extracted video features](https://github.com/facebook/C3D) for activity recognition (like fine-tuning). Sorry for that this project is for a my orginal video dataset that is not public. Actually, the model was accurate and showed some good results. But I think that is not proper because of the extreamly-low variation of the dataset. The training data are 16-frame short clips, extracting multiple ones from a video.
